html-tables/index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Tables</title>
+  </head>
+  <body>
+    <table>
+      <h1>Student Grade Table</h1>
```

Comment:
You want the title to be outside of the table

Reply:
ah okay I got confused with the caption I believe 
got it!